src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -22,33 +27,58 @@
     private BookRepository bookRepository;
 
     @GetMapping
+    @ApiOperation(value = "Return all users", response = User.class)
```

Comment:
Same here about informing that the response is a collection of users and not only one.